src/game/boe.town.cpp

```diff
@@ -1538,7 +1538,7 @@ void draw_map(bool need_refresh) {
 		}
 	}
 	
-	mini_map.setActive(false);
+	mini_map.setActive();
```

Comment:
Does this do something? I can't see any difference with or without this change, but maybe I'm missing something. It does seem like the change is correct, though…

Reply:
My solution to avoid flickering was to call setActive(false); do change; setActive() and there I make a mistake: I write setActive(false); do change; setActive(false), so I want to correct this mistake. Here, I suppose that the call after mini_map.display() is enough.

Reply:
I believe calling `display()` may implicitly set it active first if it isn't already, so it's probably just redundant.

Reply:
Yes, I also imagine that this is the case.